src/data/repositories/D2ApiMetadata.ts

```diff
@@ -207,6 +209,8 @@ export type D2Config = {
         periodDates: D2NamedCodeRef;
         outcomeDates: D2NamedCodeRef;
         outputDates: D2NamedCodeRef;
+        mandatoryCompanionIndicator: D2NamedCodeRef;
+        optionalCompanionIndicator: D2NamedCodeRef;
```

Comment:
note for the future: instead of using `D2NamedCodeRef` here, we can create an alias `D2Attribute` or whatever. Rationale: whenever we need to change the type, all we'll be changed, and now we'll need only to change it in one place.